_data/Chanduniverse/Chanduniverse.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Chando # No longer than 28 characters
+institution: Okemos High School, Michigan 🚩 # no longer than 58 characters
+profile_pic: soosuke.png # Name and extension of your profile picture(ex. mona.png)
```

Comment:
Incorrect file name, please do one of the following:
1. Change `soosuke.png` to `soosuke.jpg` in the markdown file.
2. Rename `soosuke.jpg` to `soosuke.png`.
I'd suggest you rename the file!